content/shmem_wait.tex

```diff
@@ -5,15 +5,15 @@
 \begin{apidefinition}
 
 \begin{C11synopsis}
-void shmem_wait(TYPE *ivar, TYPE cmp_value);
-void shmem_wait_until(TYPE *ivar, int cmp, TYPE cmp_value);
+[[deprecated]] void shmem_wait(TYPE *ivar, TYPE cmp_value);
+void shmem_wait_until(TYPE *ivar, shmem_cmp cmp, TYPE cmp_value);
 \end{C11synopsis}
 where \TYPE{} is one of the point-to-point synchronization types specified by
 Table \ref{p2psynctypes}.
 
 \begin{Csynopsis}
-void shmem_<TYPENAME>_wait(TYPE *ivar, TYPE cmp_value);
-void shmem_<TYPENAME>_wait_until(TYPE *ivar, int cmp, TYPE cmp_value);
+[[deprecated]] void shmem_<TYPENAME>_wait(TYPE *ivar, TYPE cmp_value);
```

Comment:
IIRC, this is the first routine that is deprecated outright. Others were deprecated with (more or less) a renamed replacement. I'd rather remove the `shmem_wait()` listing than add `[[deprecated]]` to all the other deprecated routines.